### ISSUE TITLE
Unhandled ASDF/PLAN:SYSTEM-OUT-OF-DATE in thread #<error printing a SB-THREAD:THREAD: #<PRINT-NOT-READABLE {10026914B3}>>: system cl-rdkafka is out of date

### ISSUE THREAD

**Issue by svetlyak40wt** [SahilKang/cl-rdkafka#72]
```
Unhandled ASDF/PLAN:SYSTEM-OUT-OF-DATE in thread #<error printing a SB-THREAD:THREAD: #<PRINT-NOT-READABLE {10026914B3}>>: system cl-rdkafka is out of date

Backtrace for: #<SB-THREAD:THREAD tid=10 "Kafka Producer" RUNNING {100214A8D3}>
0: (SB-DEBUG::DEBUGGER-DISABLED-HOOK #<ASDF/PLAN:SYSTEM-OUT-OF-DATE {10026773C3}> #<unused argument> :QUIT T)
1: (SB-DEBUG::RUN-HOOK *INVOKE-DEBUGGER-HOOK* #<ASDF/PLAN:SYSTEM-OUT-OF-DATE {10026773C3}>)
2: (INVOKE-DEBUGGER #<ASDF/PLAN:SYSTEM-OUT-OF-DATE {10026773C3}>)
3: (ERROR #<ASDF/PLAN:SYSTEM-OUT-OF-DATE {10026773C3}>)
4: (CL-RDKAFKA::MAKE-CONF-FROM-PLIST ("bootstrap.servers" "broker:9092" "enable.idempotence" "true"))
5: ((:METHOD CL-RDKAFKA::MAKE-CONF (LIST)) ("bootstrap.servers" "broker:9092" "enable.idempotence" "true")) [fast-method]
Unhandled ASDF/PLAN:SYSTEM-OUT-OF-DATE in thread #<error printing a SB-THREAD:THREAD: #<PRINT-NOT-READABLE {10026A6523}>>: system cl-rdkafka is out of date

```

This errors happens when I run a binary built from lisp sources using Roswell. Not checked on binaries build with asdf:make-system.

I lack of time right now, because participating in a hackathon, but will try to make reproducable minimal example later.

**Comment by jfrikker-shibumi** [comment]
I'm having the same problem, also in make-conf-from-plist. I suspect the problem is that `(asdf:find-system 'cl-rdkafka)` is being called from `with-set-keyval`; I'm not sure how that behaves when the quicklisp registry has been wiped away.

Edit: I was able to work around this by calling `(asdf:register-immutable-system "cl-rdkafka")` in my .ros file.

**Comment by SahilKang** [comment]
thanks, I've released `v1.2.2` which includes #73 with this fix
